api/auth.py

```diff
@@ -327,10 +356,82 @@ def post(self):
                 'message': 'Error, user does not exist.',
             }
             return make_response(jsonify(responseObject)), 401
+
+
+class RoomReserved(MethodView):
+    """
+    User Resource
+    """
+    from service.reserve_room_service import get_reservation, get_meditatoin_room_by_id, get_room_reserved_by_id, get_reservation_by_id, add_room_reserved_return_id, add_reservation_return_id
+
```

Comment:
Check out how this decorator should be used to authorize the request with auth token: https://github.com/ssw555tm012021Fall/halcyon-api/blob/main/api/auth.py#L158
After this, you will only need to implement the logic for API.
Check this as well for how it should be used: https://github.com/ssw555tm012021Fall/halcyon-api/blob/main/api/rooms.py#L14